docs/namespaced-broker-resources.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Using Namespaced Broker Resources
+layout: docwithnav
+---
+
+# Cluster-Scoped vs Namespace-Scoped Broker Resources
+
+Service Catalog enables service brokers to be registered in two manners: as a cluster-scoped resource or as a namespace-scoped resource. As a user of service catalog, you might use these approaches to accomplish different goals, like providing a common set of service broker resources to all users or extending your existing role based access control (RBAC) policies to the lifecycle of service broker resources. This document will explain some use cases for namespace-scoped resources and how to accomplish them using Service Catalog.
+
+## Possible Use Cases
+
+When using `ClusterServiceBroker` and associated `ClusterServiceClass` and `ClusterServicePlan` resources, service broker resources existing within the cluster as cluster-scoped resources. This means that you are limited in how you can apply RBAC and you can only have a single instance of that resource for a given identifier. As an example, if the service broker you are registering has fixed class and plan identifiers, you will be limited to one instance of the broker. With namespace-scoped brokers, however, the `ServiceBroker`, along with the `ServiceClass` and `ServicePlan` resources are scoped to a particular namespace. This allows for some more complicated use cases that were not possible with the cluster-scoped broker resources. 
```

Comment:
> service broker resources existing within the cluster as cluster-scoped resources

I think the verb here may be wrong but I'm not quite sure what you intended to say. Maybe "service broker resources, such as classes and plans, are created as cluster-scoped resources"?

Reply:
Also I suggest saying "advanced use cases" instead of "complicated" so it sounds less scary. 😁